frontend/src/metabase/home/components/EmbedHomepage/Badge.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
```

Comment:
We may use something from metabase/ui in the future for this: https://metaboat.slack.com/archives/C057WD5L0JG/p1726586842672059

I just did it as simply as possible to do what the current design needed